libraries/botbuilder-core/src/middlewareSet.ts

```diff
@@ -77,6 +78,15 @@ export class MiddlewareSet implements Middleware {
         this.use(...middlewares);
     }
 
+    [Symbol.iterator]() {
```

Comment:
Why is this necessary? I don't see any iteration through middlewares?

Reply:
Yes, it is not necessary. The first implementation was iterating through middlewareset to get TelemetrtyMiddleWare. Now, it is changed to set to turnState.